jlm/llvm/opt/alias-analyses/PointerObjectSet.cpp

```diff
@@ -463,6 +592,172 @@ PointerObjectConstraintSet::GetConstraints() const noexcept
   return Constraints_;
 }
 
+size_t
+PointerObjectConstraintSet::SolveUsingWorklist()
+{
+  size_t numWorkItems = 0;
```

Comment:
Could you move this directly over the main loop. It is used there for the first time. Currently, you have ca. 100 lines of code between its definition and the first usage.